src/index.html

```diff
@@ -2,12 +2,71 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./styles/index.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <img
+        src="images/imac.jpeg"
+        alt="APPLE A1419 iMac 27 inches"
+        class="card__product-image"
+      >
+
+      <h2 class="card__header">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h2>
+
+      <div class="card__product-code-box">
+        <p class="card__text card__text--color--secondary card__text--fz--s">
```

Comment:
to define modifier in BEM we use `--` so it is a bit weird when we have name like `card__text--color--secondary`
```suggestion
        <p class="card__text card__text--color-secondary card__text--fz-s">
```

Reply:
OMG, what a silly mistake!!!